nodestream_akamai/edns.yaml

```diff
@@ -0,0 +1,49 @@
+- implementation: nodestream_akamai.edns:AkamaiEdnsExtractor
+  arguments:
+    base_url: !config 'base_url'
+    client_token: !config 'client_token'
+    client_secret: !config 'client_secret'
+    access_token: !config 'access_token'
+    account_key: !config 'account_key'
+- implementation: nodestream.interpreting:Interpreter
+  arguments:
+    before_iteration:
+      - type: relationship
+        relationship_type: RECORD_OF
+        node_type: AkamaiEdnsZone
+        node_key: 
+          zone: !jmespath 'zone'
+        node_properties:
+          type: !jmespath 'type'
+          contract_id: !jmespath 'contractId'
+          alias_count: !jmespath 'aliasCount'
+          sign_and_serve: !jmespath 'signAndServe'
+    iterate_on: !jmespath 'recordsets[*]'
+    interpretations:
+      - type: source_node
+        node_type: AkamaiEdnsRecordSet
+        normalization:
+          do_remove_trailing_dots: true
+        key:
+          name: !jmespath 'name'
+        properties:
+          type: !jmespath 'type'
+          ttl: !jmespath 'ttl'
+      - type: relationship
+        node_type: Endpoint
```

Comment:
@zprobst Should this be a different Node type?  I looked through this and it feels like it should in fact be an Endpoint, just curious your opinion here.